compute/kubernetes/api-cli/index.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 meta:
-  title: API/CLI Documentation
+  title: Kubernetes Kapsule - API/CLI Documentation
```

Comment:
For this and the other kubernetes titles, aren't the documentation sections (API/CLI, How tos, quickstart) etc for Kubernetes generally, not specifically for Kapsule or Kosmos?

So I'd say it should just be "Kubernetes - API/CLI Documentation" or else "Scaleway Managed Kubernetes - bla bla" or "Scaleway Kubernetes - bla bla" or "Kubernetes Kosmos & Kapsule - bla bla"